README.md

```diff
@@ -6,46 +6,91 @@ schedgen
 
 This is a schedule image generator for streamer teams!
 
-How to use
-----------
+Software Requirements
+---------------------
+- [poetry](https://python-poetry.org): package manager (it should install summon as dev dependency)
+- Run `fc-match <FontName>` to get OS-hosted font file (change `<FontName>` for a font name of your choice)
 
-First of all, create a style file in TOML.
 
-Example:
+Usage
+-----
+
+```bash
+schedgen [OPTION]... WEEKDAY STREAMER...
+```
+
+Generates schedule announcement for streamers (check [example](./example.png)).
+
+Where OPTION may be one of following options:
+
+| OPTION        | Description                                             |
+|---------------|---------------------------------------------------------|
+| --background  | Background image relative path                          |
+| --config-file | config toml file relative path, default='schedgen.toml' |
+| --output      | output png image relative path, default='generated.png' |
+
+Pass WEEKDAY as a single-word string, like "tuesday" (don't include quotes)
+
+Pass each STREAMER as follows: "NICKNAME;HH:MM", where NICKNAME is a streamer nickname, like "jo3star".
+HH:MM will respect 24-hour format.
+
+For example, for scheduling "jo3star" every tuesday 9pm pass STREAMER as "jo3star;21:00"
+
+
+Example
+-------
+
+First of all, create a toml file (by default, schedgen will use `schedgen.toml`)
+Below toml example uses `demo` directory resources
 
 ```toml
 [schedgen.style]
-weekday_font = { file = "Hack-Bold.ttf", size = 50 }
-schedule_height = 458
+weekday_font = { file = "iosevka-custom-regular.ttf", size = 50 }  # Day of Week's font (main header)
+schedule_height = 458  # Result schedule image height, in px
 schedule_y = 87  # from the top
+weekday_y = 32  # from the top
 
 [schedgen.entry_style]
 stroke_width = 7
-max_height = 123
-min_spacing = 7
-width = 285
+max_height = 120
+min_spacing = 7  # min_spacing
```

Comment:
The comment is just the name of the variable.

Reply:
oh, sorry. I forgot this